sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_GetAndListOperations.md

```diff
@@ -31,45 +31,45 @@ Uri trainingFileUri = new Uri("<trainingFileUri>");
 BuildModelOperation operation = await client.BuildModelAsync(WaitUntil.Completed, trainingFileUri, DocumentBuildMode.Template);
 
 // List the first ten or fewer operations that have been executed in the last 24h.
-AsyncPageable<ModelOperationInfo> modelOperations = client.GetOperationsAsync();
+AsyncPageable<DocumentModelOperationSummary> operationSummaries = client.GetOperationsAsync();
 
 string operationId = string.Empty;
 int count = 0;
-await foreach (ModelOperationInfo modelOperationInfo in modelOperations)
+await foreach (DocumentModelOperationSummary operationSummary in operationSummaries)
```

Comment:
Are we updating this also in the swagger @HarshaNalluru ?

Reply:
Yes :)

Reply:
Thanks!